.github/workflows/release-package.yml

```diff
@@ -1,22 +1,14 @@
-name: Release & Publish Package
+name: Publish to GPR & Publish Github Release
```

Comment:
What is GPR? 👀

Reply:
Github pre-release?

Reply:
😅 I thought it was github package registry, but now I'm not sure. it was the name of the job